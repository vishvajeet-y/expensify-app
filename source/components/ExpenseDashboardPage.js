import React from 'react'
import ExpenseListFilters from './ExpenseListFilters'
import ExpenseList from './ExpenseList'
import ExpenseSummary from './ExpenseSummary'
const ExpenseDashboardPage=()=>(
    <div>
    <ExpenseSummary />
    <ExpenseListFilters/>
    <ExpenseList/>
    </div>
)
export default ExpenseDashboardPage