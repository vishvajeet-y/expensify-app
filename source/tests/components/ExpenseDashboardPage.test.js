import React from 'react'
import {shallow} from 'enzyme'
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage'

test('should test for Dashboard Page',()=>{
    const wrapper=shallow(<ExpenseDashboardPage/>)
    expect(wrapper).toMatchSnapshot()
})