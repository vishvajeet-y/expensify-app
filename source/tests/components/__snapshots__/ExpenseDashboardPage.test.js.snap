// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test for Dashboard Page 1`] = `
<div>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
